app/models/wpcc/contributions_calendar.rb

```diff
@@ -0,0 +1,56 @@
+module Wpcc
+  class ContributionsCalendar
+    attr_reader :eligible_salary,
+                :salary_frequency,
+                :employee_percent,
+                :employer_percent
+
+    PERIODS_FILE = Wpcc::Engine.root.join('config', 'contribution_periods.yml')
+    PERIODS = YAML.load_file(PERIODS_FILE)
+
+    def initialize(eligible_salary:,
+                   employee_percent:,
+                   employer_percent:,
+                   salary_frequency:)
+
+      @eligible_salary = eligible_salary
+      @employee_percent = employee_percent
+      @employer_percent = employer_percent
+      @salary_frequency = salary_frequency
+    end
+
+    def schedule
```

Comment:
@Tr4pSt3R good comment. That is something that will be addressed when we do the views. This pr is solely about the logic, and we should keep it as small and discrete as possible.